cmd/node/config/external.toml

```diff
@@ -0,0 +1,7 @@
+[Explorer]
```

Comment:
Should merge these 2 sections: move the "Explorer" configs inside "ElasticSearchConnector"

Reply:
done